### DISCUSSION TITLE
Only Allowing Search of Amazon and Bestbuy

### DISCUSSION THREAD

**Discussion by MJB1113** [jef/streetmerchant#1614] [Help]
I have an issue in the case that whenever I try to use streetmerchant, I am only able to search amazon and BestBuy. Streetmerchant worked properly the first time I used it and let me search all stores that I had selected but now only lets me use these two. Here is my .env file:

** All configuration variables are optional **
Copy this file and name it as dotenv (without backticks)
Read https://jef.codes/streetmerchant/getting-started/#customization for help on customizing this file
########################################################################################################

ASCII_BANNER=
ASCII_COLOR=
AUTO_ADD_TO_CART=true
BROWSER_TRUSTED=
COUNTRY=USA
DESKTOP_NOTIFICATIONS=true
DISCORD_NOTIFY_GROUP=
DISCORD_NOTIFY_GROUP_3060TI=true
DISCORD_NOTIFY_GROUP_3070=true
DISCORD_NOTIFY_GROUP_3080=true
DISCORD_NOTIFY_GROUP_3090=true
DISCORD_NOTIFY_GROUP_CORSAIR_SF=
DISCORD_NOTIFY_GROUP_RX6800=
DISCORD_NOTIFY_GROUP_RX6800XT=
DISCORD_NOTIFY_GROUP_RX6900XT=
DISCORD_NOTIFY_GROUP_RYZEN5600=
DISCORD_NOTIFY_GROUP_RYZEN5800=
DISCORD_NOTIFY_GROUP_RYZEN5900=
DISCORD_NOTIFY_GROUP_RYZEN5950=
DISCORD_NOTIFY_GROUP_SONYPS5C=
DISCORD_NOTIFY_GROUP_SONYPS5DE=
DISCORD_NOTIFY_GROUP_XBOXSX=
DISCORD_NOTIFY_GROUP_XBOXSS=
DISCORD_NOTIFY_GROUP_TEST=
DISCORD_WEB_HOOK=https://discord.com/api/webhooks/794820191489228821/4f5toLJBdHc6YfTPN65fMa1cPDLByG9ozqNSiqfg5B3GHcTgX-i9F9Y1iWsG6WmsiDVw
EMAIL_PASSWORD=
EMAIL_TO=
EMAIL_USERNAME=
HEADLESS=
IN_STOCK_WAIT_TIME= 600
LOG_LEVEL=
LOW_BANDWIDTH=
MAX_PRICE_SERIES_3060TI=650
MAX_PRICE_SERIES_3070=750
MAX_PRICE_SERIES_3080=850
MAX_PRICE_SERIES_3090=1800
MAX_PRICE_SERIES_CORSAIR_SF=
MAX_PRICE_SERIES_RX6800=
MAX_PRICE_SERIES_RX6800XT=
MAX_PRICE_SERIES_RX6900XT=
MAX_PRICE_SERIES_RYZEN5600=
MAX_PRICE_SERIES_RYZEN5800=
MAX_PRICE_SERIES_RYZEN5900=
MAX_PRICE_SERIES_RYZEN5950=
MAX_PRICE_SERIES_SONYPS5C=
MAX_PRICE_SERIES_SONYPS5DE=
MAX_PRICE_SERIES_XBOXSS=
MAX_PRICE_SERIES_XBOXSX=
MAX_PRICE_SERIES_TEST=
MICROCENTER_LOCATION=
MQTT_BROKER_ADDRESS=
MQTT_BROKER_PORT=
MQTT_CLIENT_ID=
MQTT_PASSWORD=
MQTT_QOS=
MQTT_TOPIC=
MQTT_USERNAME=
NVIDIA_ADD_TO_CART_ATTEMPTS=150
NVIDIA_SESSION_TTL=60
OPEN_BROWSER=true
PAGE_BACKOFF_MIN=
PAGE_BACKOFF_MAX=
PAGE_SLEEP_MIN=
PAGE_SLEEP_MAX=
PAGE_TIMEOUT=
PAGERDUTY_INTEGRATION_KEY=
PAGERDUTY_SEVERITY=
PHILIPS_HUE_API_KEY=
PHILIPS_HUE_CLOUD_ACCESS_TOKEN=
PHILIPS_HUE_CLOUD_CLIENT_ID=
PHILIPS_HUE_CLOUD_CLIENT_SECRET=
PHILIPS_HUE_CLOUD_REFRESH_TOKEN=
PHILIPS_HUE_LAN_BRIDGE_IP=
PHILIPS_HUE_LIGHT_COLOR=
PHILIPS_HUE_LIGHT_IDS=
PHILIPS_HUE_LIGHT_PATTERN=
PHONE_CARRIER=tmobile
PHONE_NUMBER=
PLAY_SOUND=
PROXY_ADDRESS=
PROXY_PROTOCOL=
PROXY_PORT=
PUSHBULLET=
PUSHOVER_EXPIRE=
PUSHOVER_RETRY=
PUSHOVER_TOKEN=
PUSHOVER_USER=
PUSHOVER_PRIORITY=
SCREENSHOT=true
SHOW_ONLY_BRANDS=
SHOW_ONLY_MODELS=
SHOW_ONLY_SERIES=
SLACK_CHANNEL=
SLACK_TOKEN=
SMTP_ADDRESS=
SMTP_PORT=
STORES="amazon,amd,adorama,antonline,bestbuy,asus,bandh,corsair,evga,gamestop,newegg,nvidia,officedepot"
TELEGRAM_ACCESS_TOKEN=
TELEGRAM_CHAT_ID=
TWILIO_ACCOUNT_SID=
TWILIO_AUTH_TOKEN=
TWILIO_FROM_NUMBER=
TWILIO_TO_NUMBER=
TWITCH_ACCESS_TOKEN=
TWITCH_CHANNEL=
TWITCH_CLIENT_ID=
TWITCH_CLIENT_SECRET=
TWITCH_REFRESH_TOKEN=
TWITTER_ACCESS_TOKEN_KEY=
TWITTER_ACCESS_TOKEN_SECRET=
TWITTER_CONSUMER_KEY=
TWITTER_CONSUMER_SECRET=
TWITTER_TWEET_TAGS=
WEB_PORT=

**Comment by jsauve** [comment]
Remove the quotes

**Comment by xedensleavesx** [comment]
I also have the same issue, this is what my dotenv file looks like 

DISCORD_WEB_HOOK=https://discord.com/api/webhooks/453296936888434688/M6uo4tYL4iNAP83-ulBBlW8RxdCHHzGPbL0UIWhJquZ81fGUVDtPX3osqn1Jtc5Gx034
OPEN_BROWSER=
PAGE_SLEEP_MIN=45000
PAGE_TIMEOUT=60000
PLAY_SOUND=
SHOW_ONLY_BRANDS=microsoft,sony
SHOW_ONLY_SERIES=sonyps5c,sonyps5de,xboxss
STORES=amazon-de,amazon-de-warehouse,amazon-es,amazon-fr,amazon-it,amazon-nl

Can anyone help ?
